content/docs/command-reference/pull.md

```diff
@@ -124,7 +124,8 @@ used to see what files `dvc pull` would download.
 
 - `--glob` - allows pulling files and directories that match the
   [pattern](https://docs.python.org/3/library/glob.html) specified in `targets`.
-  Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`
+  Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`. Only
+  matches against files present in the workspace.
```

Comment:
I'm not sure I understand this tbh :) (I don't know the context where it comes from)

Reply:
Specifically for pull this seems more like a bug than something to document. What good is it to "pull" something that's already present?

Reply:
Someone else may have pushed changes for files that I have in my workspace, and I want to pull the latest version of those specific files, but not everything in the entire workspace.

Reply:
OK to note this then:
```suggestion
  works with file names that are already in the workspace (pull their latest versions).
```
But is it also a bug or important enhancement to address? If we can `dvc ls` we should be able to match `dvc pull` to any tracked file mentioned in .dvc or dvc.yaml files, IMO.